src/components/HomePage.vue

```diff
@@ -211,17 +212,35 @@
             </div>
           </div>
           <div class="table_header">
-            <h4>Top 100 VN</h4>
+            <v-tabs v-model="tab" align-with-title>
+              <v-tabs-slider color="yellow"></v-tabs-slider>
+              <v-tab> Songs </v-tab>
+              <v-tab> Albums </v-tab>
+            </v-tabs>
             <span>Show all</span>
           </div>
           <div class="overflow-y-scroll h-50 my-1">
-            <v-data-table
-              :headers="headers"
-              @click:row="changeSong"
-              :items="desserts"
-              hide-default-footer
-              class="songsList"
-            ></v-data-table>
+            <v-tabs-items v-model="tab">
+              <v-tab-item :key="songs">
+                <v-data-table
+                  :headers="headers"
+                  :search="search"
+                  @click:row="changeSong"
+                  :items="desserts"
```

Comment:
why is the term `desserts` used for songs here and below?

Reply:
Well, that was a typo. We are used to clicking enter on the suggested word that pops up in VS code..so just pressed enter and proceeded with it...right now, I've made changes to those. Thanks for the update!! I've replaced desserts with songs...Maybe I was in the mood to eat some desserts lol!!!